docs/changelog.rst

```diff
@@ -1,9 +1,13 @@
 Changelog
 ~~~~~~~~~
 
+.. currentmodule:: pytket.extensions.qiskit
```

Comment:
Is this deliberate? I'm unfamiliar with the syntax

Reply:
Yeah, basically it means that the refernces to classes and functions don't need to be fully qualified. The link is also generated

i.e.

```
:py:class:`IBMQBackend`
```

vs 

```
:py:class:`~pytket.extensions.qiskit.IBMQBackend`
```